app/src/test/java/com/futurice/freesound/feature/search/SoundItemViewModelTest.kt

```diff
@@ -313,12 +320,36 @@ class SoundItemViewModelTest {
 
     @Test
     fun openDetails_openSoundViaNavigator() {
-        SoundItemViewModel(TEST_SOUND, navigator, audioPlayer, freeSoundApiService)
+        SoundItemViewModel(TEST_SOUND, navigator, audioPlayer, freeSoundApiService, tabController)
                 .openDetails()
 
         verify(navigator).openSoundDetails(eq(TEST_SOUND))
     }
 
+    @Test
+    fun hasGeotag() {
```

Comment:
With the test method name, you should include the scenario that you are testing in the name.  The general form is like: `<assert condition>_<input/state condition>`